src/js/components/container/Home.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import Canvas from './Canvas';
+import AppContainer from '../presentational/styled/AppContainer';
+import Header from '../presentational/styled/Header';
+import ButtonPanel from './ButtonPanel';
+import CanvasContainer from '../presentational/styled/CanvasContainer';
+import LoginForm from './LoginForm';
+import LeftGutter from '../presentational/styled/LeftGutter';
+import RightGutter from '../presentational/styled/RightGutter';
+
+
+
+const colorInit = "#ff826f";
+const API_POST_URL = () => `http://localhost:3000/api/whiteboard/add`;
+
+const Home = ({match}) => {
+  const [color, setColor] = useState(colorInit);
+  const [imageData, setImageData] = useState("");
+
+  const postImageData = imgData => {
+    axios.post(API_POST_URL(), { URI: imgData})
+      .then(response => {
+        // where we'll need to add routing logic or ui logic to display whiteboard id 
+        console.log(response);
+      })
+      .catch(error => {
+        console.log(error);
+      });
+  }
+
+  return (
+    <AppContainer 
+      className="container"
+    >
+      <Header color={color}>Whiteboard++</Header>
```

Comment:
So we moved this from app.js to Home.js
What is the logic behind needing to do so? I realize this is probably the way to do things, but I just want to know how we arrived there I suppose.

Reply:
So that's a really great question. As we develop the site out more and more our routing won't route us to different pages like you might typically expect but instead we'll start routing on components. 

One of the major benefits here is that visually we'll be able to tell exactly what components will be rendered and also make it easier to reuse code and apply similar formatting across the site. 

You can check out the following site to learn more about react router if you're interested. 

https://reacttraining.com/react-router/web

Reply:
Ahhh thank you so much! That little example helped so much in relating what our end-goal will be to accomplish. Simple to use it seems, yet very efficient/powerful if you can understand the over-arching concept/premise (which at the moment I don't fully grasp as of yet, but once we continue to build and tinker hopefully i'll understand more).